specs/signature-envelope-cose.md

```diff
@@ -131,8 +131,6 @@ Notary v2 supports the following unprotected header parameters:
 
 Note: `<<` and `>>` are used to notate the CBOR byte string resulting from encoding the data item.
 
-- **[`x5chain`](https://datatracker.ietf.org/doc/html/draft-ietf-cose-x509-08#section-2)** (*array of bstr*): This REQUIRED parameter (label `33` by [IANA](https://www.iana.org/assignments/cose/cose.xhtml#header-parameters)) contains the ordered list of X.509 certificate or certificate chain ([RFC5280](https://datatracker.ietf.org/doc/html/rfc5280)) corresponding to the key used to digitally sign the COSE. The certificate chain is represented as an array of certificate, each certificate in the array is DER encoded and then wrapped in a CBOR byte string. The certificate containing the public key corresponding to the key used to digitally sign the COSE MUST be the first certificate, followed by the intermediate and root certificates in the correct order. Refer [*Certificate Chain* unsigned attribute](signature-specification.md#unsigned-attributes) for more details. Optionally, this header can be presented in the protected header.
-  - **TODO** update signature specification to allow chains in protected header
```

Comment:
Can we only remove TODO line because as per TODO we want to keep supporting certificate-chain as unsigned attribute and it addition to that we also want to support certificate-chain as signed attribute.